From b91e39701c7a71f229be34dcf078579d37367436 Mon Sep 17 00:00:00 2001
From: Dan Lorenc <dlorenc@chainguard.dev>
Date: Sat, 24 Dec 2022 12:15:14 -0500
Subject: [PATCH] Disable protected mode.

See https://github.com/docker-library/redis/blob/19f345b2056588c009396bcfc22ba9241a669491/Dockerfile-alpine.template#L43
for more context - we disable protected mode because it's not needed
in a docker image and we can't do this via a config option.

---
 src/config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.c b/src/config.c
index 78553b758..8547d76c2 100644
--- a/src/config.c
+++ b/src/config.c
@@ -3013,7 +3013,7 @@ standardConfig static_configs[] = {
     createBoolConfig("daemonize", NULL, IMMUTABLE_CONFIG, server.daemonize, 0, NULL, NULL),
     createBoolConfig("io-threads-do-reads", NULL, DEBUG_CONFIG | IMMUTABLE_CONFIG, server.io_threads_do_reads, 0,NULL, NULL), /* Read + parse from threads? */
     createBoolConfig("always-show-logo", NULL, IMMUTABLE_CONFIG, server.always_show_logo, 0, NULL, NULL),
-    createBoolConfig("protected-mode", NULL, MODIFIABLE_CONFIG, server.protected_mode, 1, NULL, NULL),
+    createBoolConfig("protected-mode", NULL, MODIFIABLE_CONFIG, server.protected_mode, 0, NULL, NULL),
     createBoolConfig("rdbcompression", NULL, MODIFIABLE_CONFIG, server.rdb_compression, 1, NULL, NULL),
     createBoolConfig("rdb-del-sync-files", NULL, MODIFIABLE_CONFIG, server.rdb_del_sync_files, 0, NULL, NULL),
     createBoolConfig("activerehashing", NULL, MODIFIABLE_CONFIG, server.activerehashing, 1, NULL, NULL),
-- 
2.37.1 (Apple Git-137.1)

